### ISSUE TITLE
Overfitting problem

### ISSUE THREAD

**Issue by TianchaoHuo** [koyeongmin/PINet#22]
Hello, firstly thank you for your great work. After I look through the code and the data set (tuSimple), I am confused that if the overfitting problem would occur ? Because i notice that you train the model with 1000 epoch but the data set only has 3000 images around. whether the training epoch number is too large or not?  And i think the test set of tuSimple is similar to train set and therefore, whether this overfitting problem affects the performance on test set or not?

**Comment by koyeongmin** [comment]
In test set of tuSimple, the model works well, because test set is similar to training set.(the provided pre train model don't work well in other environmen. If you want to apply to other dataset, you need to train again on your dataset)
